comms/minicom/Portfile

```diff
@@ -27,16 +28,21 @@ depends_build       port:pkgconfig
 depends_lib         port:gettext \
                     port:ncurses
 
-depends_run         port:kermit \
-                    port:lrzsz
+depends_run         port:lrzsz
 
 post-patch {
     reinplace "s|/usr/local|${prefix}|g" ${worksrcpath}/src/rwconf.c
     reinplace "s|/usr/bin/ascii-xfr|${prefix}/bin/ascii-xfr|g" ${worksrcpath}/src/rwconf.c
 }
 
-configure.args      --enable-lock-dir=/tmp \
-                    --enable-kermit=${prefix}/bin
+configure.args      --enable-lock-dir=/tmp
+
+variant kermit description {Builds minicom with kermit support} {
+    depends_run-append      port:kermit
+    configure.args-append   --enable-kermit=${prefix}/bin
```

Comment:
Given this has configure arguments, should this only be a runtime dependency?

Reply:
I'm not sure if Kermit is needed during the build process - I honestly haven't thought about that. Will investigate that later and report back.

Reply:
> Given this has configure arguments, should this only be a runtime dependency?

so it seems that minicom compiles fine with `./configure --enable-lock-dir=/tmp --enable-kermit=/opt/local/bin` while kermit is not installed. But I'm fine with a change to a lib-dependency

Reply:
The real test is to see if kermit is listed in `otool -L` output of the `minicom` executable.

Reply:
You can leave it as a runtime dependency for now.